pkg/server/ca/ca_test.go

```diff
@@ -59,12 +59,9 @@ func (s *CATestSuite) SetupTest() {
 	s.logHook = logHook
 
 	s.ca = NewCA(Config{
-		Log:     log,
-		Metrics: telemetry.Blackhole{},
-		TrustDomain: url.URL{
-			Scheme: "spiffe",
-			Host:   "example.org",
-		},
+		Log:         log,
+		Metrics:     telemetry.Blackhole{},
+		TrustDomain: spiffeid.RequireTrustDomainFromString("spiffe://example.org"),
```

Comment:
We use "example.org" trust domain in lots of spots in these tests. There is a lot of repetition here. Maybe we can declare the td at the package level and then use the `ID()` method where we want the ID of the trust domain...

```
// at package scope
var td =  spiffeid.RequireTrustDomainFromString("example.org")

...

// wherever we need the ID of the trust domain
tdID := td.ID()
```